assets/src/domain/eventEditor/ui/datetimes/datesList/cardView/DateCardList.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { CSSProperties } from 'react';
+
+import DateCard from './DateCard';
+
+const listStyle: CSSProperties = {
+	display: 'flex',
+	flexFlow: 'row wrap',
+	justifyContent: 'space-between',
+	width: '100%',
+};
```

Comment:
move this into csss file plz

Reply:
Extracted: https://github.com/eventespresso/event-espresso-core/pull/2431/commits/e2a483902c0f0b7d513211e0c7e458716e7a209c